src/couch_replicator_httpc_pool.erl

```diff
@@ -55,114 +58,113 @@ release_worker(Pool, Worker) ->
 
 init({Url, Options}) ->
     process_flag(trap_exit, true),
-    State = #state{
+    State0 = case couch_replicator_dns:lookup(Url) of
+        {ok, {Data, TTL}} ->
+            timer:send_after(TTL * 1000, refresh_dns),
```

Comment:
This seems the right thing to do but is the TTL ever actually relevant for us? Do we have a different, distinct pattern maybe when we would expect DNS updates? Just wondering.